README.md

```diff
@@ -1,6 +1,17 @@
-# `check-protocols`
+# Check Protocols [![CircleCI](https://circleci.com/gh/Originate/check-protocols.svg?style=svg)](https://circleci.com/gh/Originate/check-protocols) [![Waffle.io - Columns and their card count](https://badge.waffle.io/Originate/check-protocols.svg?columns=all)](https://waffle.io/Originate/check-protocols)
```

Comment:
Why do you want to add the circleci badge? This should always be green and is a bit redundant.

Reply:
i think it's a nice way to indicate to people that "we have CI set up for this repo, and we care about the tests passing".

Reply:
Sounds good.